### PR TITLE
[community]  added member profile

### LINKED ISSUE #3836: [Community] Member Profile: Deepak Dinesh
**Current Behavior**

@deepakdinesh1123  has been a consistent contributor and community member.

**Desired Situation**
Let's recognize @deepakdinesh1123  as a contributor and community member by creating a profile on https://layer5.io/community/members.

- GitHub: @deepakdinesh1123 
- Twitter: 
- LinkedIn: 
- Link to profile picture:

A detailed explanation on how to set up a community member profile can be found in the [CONTRIBUTING.md](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

---

**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Join the [Layer5 Community](http://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by sidharthviz**
**Description**

This PR fixes #3836 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 07bbf5bd398f84165834fdf7ae1b06b23cc01bf2 at: https://63ff4878a11618006bb96c14--layer5.netlify.app

File: .gitpod.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+# Please adjust to your needs (see https://www.gitpod.io/docs/introduction/learn-gitpod/gitpod-yaml)
+# and commit this file to your remote git repository to share the goodness with others.
+
+# Learn more from ready-to-use templates: https://www.gitpod.io/docs/introduction/getting-started/quickstart
+
+tasks:
+  - init: npm install && npm run build && make
+    command: npm run start
+
+
```
Comment (Chadha93) [inline .gitpod.yml:11]: @sidharthviz Please remove this file.

File: src/collections/members/deepak dinesh/index.mdx [hunk 2]
```diff
@@ -0,0 +1,12 @@
+---
+name: Deepak dinesh
+position: Contributor
+image_path: ./deepak.png
+github:  deepakdinesh1123 
+linkedin: https://www.linkedin.com/in/deepak-dinesh-6444b71a0/
+twitter: https://twitter.com/LordDaemon13
+location: Karnataka, India
+bio:
```
Comment (Chadha93) [inline src/collections/members/deepak dinesh/index.mdx:9]: @deepakdinesh1123 will you toss in the bio here?

**Comment by Chadha93** [comment]
I don't see Deepak, in the build preview. @sidharthviz Will you check?

**Pull request closed without merging**